docs/src/_cloud/fees.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Cloud Fees"
+hidden: true
+---
+<div class="notification is-info">
+  <p><strong>Meltano Cloud is currently in Beta.</strong></p>
+  <p>While in Beta, functionality is not guaranteed and subject to change. <br> If you're interested in using Meltano Cloud please join our <a href="https://meltano.com/cloud/">waitlist</a>.</p>
+</div>
+
+## Meltano Cloud Fees
+
+Todo
```

Comment:
@DouweM I'm not clear why we need a docs page for this when we have `meltano.com/pricing`. But I don't want to block on it.